### ISSUE TITLE
Keyboard devices without the string "keyboard" in them are not recognized

### ISSUE THREAD

**Issue by hartkes** [kbingham/xhk#3]
Several of my keyboards do not have the string "keyboard" in their device names.  Some have shortened forms (like "KB"), some just have the manufacturer and product name, and a PS2-to-USB adapter identifies itself as a "Barcode reader".  The function identify_local_keyboard then fails.  I worked around this issue by hard coding my device's name to be acceptable, but I think a better solution would be to allow a command line parameter to specify the device number of the desired keyboard.  The device number can be found using "xinput list".

Thanks for writing and releasing this program!  It has been a life saver since I badly sprained my right thumb.
Stephen

**Comment by kbingham** [comment]
Yes, this sounds like a good idea. basic string matching is a bit fragile. Hope your thumb is feeling better now!

**Comment by cprn** [comment]
Please merge. Written with left hand only. 😂 Including the emote.
